options.go

```diff
@@ -163,27 +144,38 @@ func loadEnv(opts *Options) {
 	opts.Env.Contents = []byte(value)
 }
 
-func findManifest(filename string) ([]byte, string, bool, error) {
+func (opts *Options) findManifest() error {
+	path, found, err := findManifest(opts.Script.Name)
```

Comment:
I know that old code has it too, but why do we search manifest, by using `opts.Script.Name` ? :thinking: 

I think it could confuse a bit of future contributors and if that's, that's the right way, it's good to have a comment here

Reply:
I find it more readable to have `findManifest()` pass the source file name as a parameter, rather than `setManifest()` implicitly using the script name. This is just a subjective comment :)